src/main/scala/GameRunner.scala

```diff
@@ -0,0 +1,113 @@
+import GameHelpers._
+import GamePrompts._
+import scala.util.Random
+
+object GameRunner extends App {
+
+  def gameStart(): Unit = {
+    println()
+    println("Welcome to \u001b[31;1mScala Scrabble Practice")
+    println()
+    val name: String = promptName()
+    println()
+    println("Hello " + name + ". \u001b[0mThe Human plays against the Computer.")
+    println()
+    val level: String = promptLevel()
+
+    val letterBag = Bag.initialise()
+    val human = Player(name, Nil, Nil, false)
+    val computer = Player("The Computer", Nil, Nil, false)
+    val newGame = Game(letterBag, human, computer, level)
```

Comment:
Nice encapsulation of all the state required to run the game 👍! How would you model this if we wanted to have a 3-player game of Scrabble? Or 6 player?